src/Tribute.js

```diff
@@ -367,6 +367,40 @@ class Tribute {
             throw new Error('No active state. Please use append instead and pass an index.')
         }
     }
+
+    detach(el) {
+        if (!el) {
+            throw new Error('[Tribute] Must pass in a DOM node or NodeList.')
+        }
+
+        // Check if it is a jQuery collection
+        if (typeof jQuery !== 'undefined' && el instanceof jQuery) {
+            el = el.get()
+        }
+
+        // Is el an Array/Array-like object?
+        if (el.constructor === NodeList || el.constructor === HTMLCollection || el.constructor === Array) {
+            let length = el.length
+            for (var i = 0; i < length; ++i) {
+                this._detach(el[i])
+            }
+        } else {
+            this._detach(el)
+        }
+    }
+
+    _detach(el) {
+        this.events.unbind(el)
+        this.menuEvents.unbind(this.menu)
+
+        setTimeout(() => {
+            el.removeAttribute('data-tribute')
+            this.isActive = false
+            this.menu.remove()
```

Comment:
In my usage scenario, after the user opens the input box, it does not necessarily use '@'. At this time, if I use 'detach', an error occurs because the menu is 'undefined' at this time.
```
if (this.menu) {
  this.menu.remove()
}
```